test/livebook/session_test.exs

```diff
@@ -2092,6 +2093,92 @@ defmodule Livebook.SessionTest do
     end
   end
 
+  describe "code evaluation logging" do
+    test "logs code evaluation for regular sessions" do
+      session = start_session()
+      Session.subscribe(session.id)
+
+      {_section_id, cell_id} = insert_section_and_cell(session.pid)
+
+      unique_id = Utils.random_short_id()
+      # Use random ID to uniquely identify the source code of this cell
+      Session.set_cell_attributes(session.pid, cell_id, %{source: "# #{unique_id}"})
+
+      log =
+        capture_log(fn ->
+          Session.queue_cell_evaluation(session.pid, cell_id)
+          assert_receive {:operation, {:add_cell_evaluation_response, _, ^cell_id, _, _}}
+        end)
+
+      # Logs from other test might be captured, so we're using an unique_id
+      assert log =~ ~s(Evaluating code\n  Session mode: default\n  Code: "# #{unique_id}")
```

Comment:
Given that we plan to remove this log format, we should assert against the other one.